ecl/network/v2/load_balancers/requests.go

```diff
@@ -0,0 +1,183 @@
+		return LoadBalancerPage{pagination.LinkedPageBase{PageResult: r}}
+	})
+}
+
+// Get retrieves a specific Load Balancer based on its unique ID.
+func Get(c *eclcloud.ServiceClient, id string) (r GetResult) {
+	_, r.Err = c.Get(getURL(c, id), &r.Body, nil)
+	return
+}
+
+// CreateOpts represents the attributes used when creating a new Load Balancer.
+type CreateOpts struct {
+
+	// AvailabilityZone is one of the Virtual Server (Nova)’s availability zone.
+	AvailabilityZone string `json:"availability_zone,omitempty"`
+
+	// Description is description
+	Description string `json:"description,omitempty"`
+
+	// LoadBalancerPlanID is the UUID of Load Balancer Plan.
+	LoadBalancerPlanID string `json:"load_balancer_plan_id" required:"true"`
+
+	// Name is a human-readable name of the Load Balancer.
+	Name string `json:"name,omitempty"`
+
+	// The UUID of the project who owns the Load Balancer. Only administrative users
+	// can specify a project UUID other than their own.
+	TenantID string `json:"tenant_id,omitempty"`
+}
+
+// ToLoadBalancerCreateMap builds a request body from CreateOpts.
+func (opts CreateOpts) ToLoadBalancerCreateMap() (map[string]interface{}, error) {
+	b, err := eclcloud.BuildRequestBody(opts, "load_balancer")
+	if err != nil {
+		return nil, err
+	}
+
+	return b, nil
+}
+
+// Create accepts a CreateOpts struct and creates a new Load Balancer using the values
+// provided. You must remember to provide a valid LoadBalancerPlanID.
+func Create(c *eclcloud.ServiceClient, opts CreateOpts) (r CreateResult) {
+	b, err := opts.ToLoadBalancerCreateMap()
+	if err != nil {
+		r.Err = err
+		return
+	}
+	_, r.Err = c.Post(createURL(c), b, &r.Body, nil)
+	return
+}
+
+// UpdateOpts represents the attributes used when updating an existing Load Balancer.
+type UpdateOpts struct {
+
+	// AdminUsername is admin's username
+	AdminUsername string `json:"admin_username"`
+
+	// Description is description
+	DefaultGateway string `json:"default_gateway,omitempty"`
+
+	// Description is description
+	Description string `json:"description,omitempty"`
+
+	// LoadBalancerPlanID is the UUID of Load Balancer Plan.
+	LoadBalancerPlanID string `json:"load_balancer_plan_id,omitempty"`
+
+	// Name is a human-readable name of the Load Balancer.
+	Name string `json:"name,omitempty"`
+}
+
+// ToLoadBalancerUpdateMap builds a request body from UpdateOpts.
+func (opts UpdateOpts) ToLoadBalancerUpdateMap() (map[string]interface{}, error) {
+	b, err := eclcloud.BuildRequestBody(opts, "load_balancer")
+	if err != nil {
+		return nil, err
+	}
+
+	return b, nil
+}
+
+// Update accepts a UpdateOpts struct and updates an existing Load Balancer using the
+// values provided.
+func Update(c *eclcloud.ServiceClient, id string, opts UpdateOpts) (r UpdateResult) {
+	b, err := opts.ToLoadBalancerUpdateMap()
+	if err != nil {
+		r.Err = err
+		return
+	}
+	_, r.Err = c.Put(updateURL(c, id), b, &r.Body, &eclcloud.RequestOpts{
+		OkCodes: []int{200, 201},
```

Comment:
https://ecl.ntt.com/en/documents/api-references/network/load_balancer/load_balancer.html#id33
```suggestion
		OkCodes: []int{200},
```

Reply:
Thank you.
There are same problems in below so I will fix them:
load_balancer_syslog_servers.Update
load_balancer_interfaces.Update